src/DataModelGraph/DataModelGraph.jsx

```diff
@@ -36,15 +34,13 @@ class DataModelGraph extends React.Component {
 
     if (graph.nodes.length !== 0 && 'count' in graph.nodes[graph.nodes.length - 1]) {
       return (
-        <div style={{ position: 'relative' }}>
-          <ToggleButton>
-            <button
-              id="cd-dmg__toggle"
-              className="button-primary-white"
-              onClick={this.handleToggleClick}
-            >Toggle view
-            </button>
-          </ToggleButton>
+        <div className="data-model-graph">
+          <button
```

Comment:
do we need to add the styles from `ToggleButton`?

Reply:
`ToggleButton` only has 2 css attribute `top` and `left`, They don't have any effect for the button in this page so I directly remove them